Update JDBC instructions (#14393)
cockroachdb/docs: cockroachcloud/quickstart.md

<section class="filter-content" markdown="1" data-scope="java">

The **Connect to cluster** dialog shows information about how to connect to your cluster.

1. Select **Java** from the **Select option** dropdown.

> 1. Copy the **JDBC_DATABASE_URL** environment variable command provided and save it in a secure location.

Reviewer: ```suggestion
1. Copy the `JDBC_DATABASE_URL` environment variable command provided and save it in a secure location.
```

<section class="filter-content" markdown="1" data-scope="java">

<section class="filter-content" markdown="1" data-scope="mac">

> In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:

Reviewer: ```suggestion
In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied earlier:
```

{% include_cached copy-clipboard.html %}
~~~ shell
export JDBC_DATABASE_URL="<jdbc-connection-string>"
~~~

The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
</section>

<section class="filter-content" markdown="1" data-scope="linux">

> In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:

Reviewer: ```suggestion
In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:
```
```suggestion
In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied earlier:
```

Reviewer: GitHub has been doing a weird thing with duplicating suggestions... Ignore the empty suggestion above

{% include_cached copy-clipboard.html %}
~~~ shell
export JDBC_DATABASE_URL="<jdbc-connection-string>"
~~~

The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
</section>

<section class="filter-content" markdown="1" data-scope="windows">

> In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied in Step 2 above:

Reviewer: ```suggestion
In a terminal set the `JDBC_DATABASE_URL` environment variable to the connection string by running the command that you copied earlier:
```

{% include_cached copy-clipboard.html %}
~~~ shell
$env:JDBC_DATABASE_URL = "<jdbc-connection-string>"
~~~

The code sample uses the connection string stored in the environment variable `JDBC_DATABASE_URL` to connect to your cluster.
</section>

{{site.data.alerts.callout_success}}

> For reference information about connecting to CockroachDB with supported client drivers, see [Connect to a CockroachDB Cluster](../../docs/stable/connect-to-the-database.html).

Reviewer: ```suggestion
For information about connecting to CockroachDB with supported client drivers, see [Connect to a CockroachDB Cluster](../../docs/stable/connect-to-the-database.html).
```

Author: This text is copied from database-url-environment-variable.md
![image](https://user-images.githubusercontent.com/24593519/176732485-427dcf5d-99b8-42f3-9591-3c393702cb47.png)

and used for the node, python, and go languages already. This simply adds the same message to the java code. 
![image](https://user-images.githubusercontent.com/24593519/176732752-6eb1c876-8227-418c-88bf-770662871ef1.png)

Changing this here would require me to also change the original code.

Reviewer: no prob, fine to leave as-is!
